Explorer/Assets/DCL/SDKComponents/MediaStream/MediaFactory.cs

```diff
@@ -96,7 +95,7 @@ public bool TryAddConsumer(Entity consumerEntity, CRDTEntity videoPlayerCrdtEnti
 
             // Only TextureData contains referencing mechanism
             // Create or get it from the entity
-            if (!world.TryGet(videoPlayerEntity, out TextureData textureData))
+            if (!world.TryGet(videoPlayerEntity, out TextureData? textureData))
```

Comment:
Is the nullable notation required here? When used, according to this code, it will always be used